db/migrate/20170118054424_change_media_kind_files_to_documents.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeMediaKindFilesToDocuments < ActiveRecord::Migration
+  def up
+    execute "UPDATE media SET kind = 'document' WHERE kind = 'file'"
```

Comment:
Maybe I should also rename `kind` to `kind_value`? All other translated option sets seem to be using that convention.